.circleci/config.yml

```diff
@@ -3,46 +3,52 @@ version: 2.1
 orbs:
   hmpps: ministryofjustice/hmpps@1.1.3
   snyk: snyk/snyk@0.0.12
+  node: circleci/node@4.1.0
+
+executors:
+  integration:
+    docker:
+      - image: circleci/node:10-buster-browsers
+      - image: rodolpheche/wiremock:2.27.2-alpine
+        command: ["--port", "9091"]
+      - image: bitnami/redis:6.0
+        environment:
+          ALLOW_EMPTY_PASSWORD: "yes"
+      - image: quay.io/hmpps/hmpps-auth:latest
```

Comment:
this isn't needed - it's covered by wiremock

Reply:
removed in 275fe55